shared/js/ui/templates/shared/grade-scorecard-reasons.es6.js

```diff
@@ -35,10 +35,11 @@ function getReasons (site) {
   // only show a message if there's any blocked
   const numTrackerNetworks = site.trackerNetworks.length
   const foundOrBlocked = site.isWhitelisted || numTrackerNetworks === 0 ? 'Found' : 'Blocked'
+  const networkOrNetworks = (site.totalTrackersCount === 1) ? 'Network' : 'Networks'
   if (numTrackerNetworks) {
     reasons.push({
       modifier: 'bad',
-      msg: `${numTrackerNetworks} Tracker Networks ${foundOrBlocked}`
```

Comment:
Awesome! Thanks for doing this.

Do you think maybe this could be a shared template, since it's being repeated 3 times?

Reply:
I agree, foundOrBlocked should be shared too for that matter: it's used pretty much everywhere!